examples/c/minimal_legacy.bpf.c

```diff
@@ -0,0 +1,34 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+#define BPF_NO_GLOBAL_DATA
+#include "linux/bpf.h"
+#include "sched.h"
```

Comment:
why not `#include <linux/bpf.h>` ? Also why do we need sched.h? Can we use some system-wide UAPI header instead?

Reply:
shed.h is for pid_t,. I should have used the <> for bpf.h. 
I could just define pid_t locally if that is easier.